src/lib/EDSL.mli

```diff
@@ -288,7 +288,58 @@ val tmp_file: ?tmp_dir: string t -> string -> file
 
 (** {3 Command Line Parsing} *)
 
+(** Typed command-line parsing for your shell scripts, à la {!Prtinf.scanf}. *)
 module Command_line: sig
+  (** 
+
+     Use this module like OCaml's {!Printf.scanf} function.
+
+     - Build a command-line “format specification” using the {!Arg} module.
+     - Call the {!parse} function with an appropriately typed function.
+
```

Comment:
I would add, "Here is a potential argument specification for a shell script that downloads and unarchives them."